test/rulesets/Base/array.jl

```diff
@@ -35,3 +35,18 @@ end
     test_rrule(fill, 44.0, 4; check_inferred=false)
     test_rrule(fill, 2.0, (3, 3, 3) ⊢ DoesNotExist())
 end
+
+@testset "repeat" begin
+    test_rrule(repeat, randn(5), 3)
+    test_rrule(repeat, randn(5), 3, 3)
+    test_rrule(repeat, randn(3, 3), 2)
+    test_rrule(repeat, randn(5, 5), 2,5)
+    test_rrule(repeat, randn(5, 4, 3); fkwargs=(inner=(2, 2, 1), outer=(1, 1, 3)))
+    test_rrule(repeat, fill(4.0), 3)
+    test_frule(repeat, randn(5), 3)
+    test_frule(repeat, randn(5), 3,3)
+    test_frule(repeat, randn(3, 3), 2)
+    test_frule(repeat, randn(3, 3), 2,5)
+    test_frule(repeat, randn(5, 4, 3); fkwargs=(inner=(2, 2, 1), outer=(1, 1, 3)))
+    test_frule(repeat, fill(4.0), 3)
```

Comment:
```suggestion
    test_rrule(repeat, randn(5), 3)
    test_rrule(repeat, randn(5), 3, 3)
    test_rrule(repeat, randn(3, 3), 2)
    test_rrule(repeat, randn(5, 5), 2,5)
    test_rrule(repeat, randn(5, 4, 3); fkwargs=(inner=(2, 2, 1), outer=(1, 1, 3)))
    test_rrule(repeat, fill(4.0), 3)
    
    @testset "frule" begin
        test_frule(repeat, fill(4.0), 3)
        
        test_frule(repeat, randn(5), 3)
        test_frule(repeat, randn(5), 3, 3)
        
        test_frule(repeat, randn(3, 3), 2)
        test_frule(repeat, randn(3, 3), 2, 5)
        
        test_frule(repeat, randn(5, 4, 3); fkwargs=(inner=(2, 2, 1), outer=(1, 1, 3)))
    end
```

and similarly for the rrule to keep things tidy